README.md

```diff
@@ -395,13 +395,14 @@ OpenFaaS Cloud can use cert-manager to automatically provision TLS certificates
 
 > This feature is optional, but highly recommended
 
-Pick between the following providers for the DNS01 challenge:
+Pick between the following providers for the [DNS01 challenge](https://cert-manager.io/docs/configuration/acme/dns01/):
 
 * DigitalOcean DNS (free at time of writing)
```

Comment:
Slightly side note, but is this still free?

Reply:
Yes